src/main/java/banking/dao/AccountDAO.java

```diff
@@ -0,0 +1,155 @@
+                id = res.getInt("id");
+            }
+        } catch (SQLException e) {
+            System.out.println(e.getMessage());
+        }
+        return id;
+    }
+
+    public boolean containCard(String cardNumber) {
+        ResultSet res;
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(SELECT_CARD_NUMBER)) {
+            preparedStatement.setString(1, cardNumber);
+            res = preparedStatement.executeQuery();
+            if (res.next()) {
+                return res.getBoolean(1);
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return false;
+    }
+
+    public int login(String cardNumber, String PIN) {
+        ResultSet res;
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(SELECT_ID)) {
+            preparedStatement.setString(1, cardNumber);
+            preparedStatement.setString(2, PIN);
+            res = preparedStatement.executeQuery();
+            if (res.next()) {
+                return res.getInt("id");
+            } else {
+                return -1;
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return -1;
+    }
+
+    public int getBalance(int id) {
+        int balance = -1;
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(GET_CARD)) {
+            preparedStatement.setInt(1, id);
+            ResultSet res = preparedStatement.executeQuery();
+            assert res != null;
+            if (res.next()) {
+                balance = res.getInt("balance");
+                return balance;
+            }
+        } catch (SQLException throwables) {
+            throwables.printStackTrace();
+        }
+        return balance;
+    }
+
+    public String getNumber(int id) {
+        String number = "";
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(GET_CARD)) {
+            preparedStatement.setInt(1, id);
+            ResultSet res = preparedStatement.executeQuery();
+            assert res != null;
+            if (res.next()) {
+                number = res.getString("number");
+                return number;
+            }
+        } catch (SQLException e) {
+            e.printStackTrace();
+        }
+        return number;
+    }
+
+    public void addBalance(int id, int amount) {
+        try (PreparedStatement preparedStatement = dbManager.getConnection().prepareStatement(UPD_BALANCE)) {
+            preparedStatement.setInt(1, amount);
+            preparedStatement.setInt(2, id);
+            preparedStatement.executeUpdate();
+        } catch (SQLException ex) {
+            ex.printStackTrace();
+        }
+    }
+
+    public void sendMoney(int id, int amountSend) {
+        try (PreparedStatement preparedStatement1 = dbManager.getConnection().prepareStatement(SEND_MONEY)) {
```

Comment:
Why this field was called `preparedStatement1 `?

Reply:
fixed